src/main/java/de/zalando/aruha/nakadi/service/subscription/StreamParameters.java

```diff
@@ -35,9 +36,9 @@
     public final long commitTimeoutMillis;
 
     private StreamParameters(
```

Comment:
I agree, but the intention of this PR is only to align nulls handling, so I will not do it here.